src/test/rustdoc-ui/issue-83883-describe-lints.stdout

```diff
@@ -1,193 +1,24 @@
-    -W                            trace-macros=val -- for every macro invocation, print its name and arguments (default: no)
-    -W   translate-remapped-path-to-local-path=val -- translate remapped paths into local paths when possible (default: yes)
-    -W                        trap-unreachable=val -- generate trap instructions for unreachable intrinsics (default: use target setting, usually yes)
-    -W                        treat-err-as-bug=val -- treat error number `val` that occurs as bug
-    -W                   trim-diagnostic-paths=val -- in diagnostics, use heuristics to shorten paths referring to items
-    -W                              ui-testing=val -- emit compiler diagnostics in a form suitable for UI testing (default: no)
-    -W            uninit-const-chunk-threshold=val -- allow generating const initializers with mixed init/uninit chunks, and set the maximum number of chunks for which this is allowed (default: 16)
-    -W          unleash-the-miri-inside-of-you=val -- take the brakes off const evaluation. NOTE: this is unsound (default: no)
-    -W                                unpretty=val -- present the input source, unstable (and less-pretty) variants;
-        `normal`, `identified`,
-        `expanded`, `expanded,identified`,
-        `expanded,hygiene` (with internal representations),
-        `ast-tree` (raw AST before expansion),
-        `ast-tree,expanded` (raw AST after expansion),
-        `hir` (the HIR), `hir,identified`,
-        `hir,typed` (HIR with types for each node),
-        `hir-tree` (dump the raw HIR),
-        `mir` (the MIR), or `mir-cfg` (graphviz formatted MIR)
-    -W                        unsound-mir-opts=val -- enable unsound and buggy MIR optimizations (default: no)
-    -W                        unstable-options=val -- adds unstable command line options to rustc interface (default: no)
-    -W                       use-ctors-section=val -- use legacy .ctors section for initializers rather than .init_array
-    -W                            validate-mir=val -- validate MIR after each transformation
-    -W                                 verbose=val -- in general, enable more debug printouts (default: no)
-    -W                          verify-llvm-ir=val -- verify LLVM IR (default: no)
-    -W            virtual-function-elimination=val -- enables dead virtual function elimination optimization. Requires `-Clto[=[fat,yes]]`
-    -W                         wasi-exec-model=val -- whether to build a wasi command or reactor
+Available lint options:
+    -W <foo>           Warn about <foo>
+    -A <foo>           Allow <foo>
+    -D <foo>           Deny <foo>
+    -F <foo>           Forbid <foo> (deny <foo> and all attempts to override)
+
+
+Lint checks provided by rustc:
+
+    $NAMES  $LEVELS  $MEANINGS
+
+Lint groups provided by rustc:
+
+    $NAMES  $SUB_LINTS
+
+Lint checks provided by plugins loaded by this crate:
+
+    $NAMES  $LEVELS  $MEANINGS
```

Comment:
this ended up fixing my original complaint in https://github.com/rust-lang/rust/pull/98533#issuecomment-1172000597 :)